### PR TITLE
Add the option to open JupyterLite window in new tab

### PR DESCRIPTION
**Pull request by melissawm**
My use case is the following: I'd like to use the jupyterlite directive to open the **full page** I am visiting as a Jupytext notebook. This PR adds that option to the JupyterLite directive.  

Pending is finding the correct path for opening the current file __as an ipynb file__: since the source is an .md file, it will be converted into an .ipynb file at some point in the build process. For now, however, Sphinx can't process the directive since it will try to embed a link to the ipynb file before reading the current source .md file, and complain about the missing reference.

I am not sure if I am following the right path here so please let me know if I should approach this differently. 

cc @agriyakhetarpal

### CODE REVIEW

**Comment by melissawm** [comment]
Here is an example of this working (partially): https://melissawm.github.io/scipy/tutorial/stats/sampling.html

**Comment by martinRenou** [comment]
Thanks for contributing! 

It's deployed in the readthedocs preview: https://jupyterlite-sphinx--165.org.readthedocs.build/en/165/directives/jupyterlite.html

Changes look good to me. I was about to suggest doing the same for other directives, but when we implement https://github.com/jupyterlite/jupyterlite-sphinx/issues/114, we can make sure the `:new_tab:` parameter works well with the coming `:app:` parameter.

Leaving this open to let other people have a look.

**Comment by melissawm** [comment]
I'm happy to do a follow up for that in the future, I might need some time togo over the app parameter and understand the effects if that's alright. Thanks!

File: jupyterlite_sphinx/jupyterlite_sphinx.py [hunk 1]
```diff
@@ -107,6 +107,39 @@ def html(self):
...
+
+        options = "&".join(
+            [f"{key}={quote(value)}" for key, value in lite_options.items()]
+        )
+        self.lab_src = f'{prefix}/{app_path}{f"?{options}" if options else ""}'
+
+        super().__init__(
+            "",
+            **attributes,
+        )
```
Comment (Carreau) [inline jupyterlite_sphinx/jupyterlite_sphinx.py:133]: Maybe `rawsource` here ?
```suggestion
        super().__init__(
            rawsource,
            **attributes,
        )
```

**Review by Carreau — APPROVED** [review]

**Review by Carreau — APPROVED** [review]
All good on my side, I'll leave ~24h  in case of objections and merge after that.

Thanks @melissawm

**Comment by Carreau** [comment]
Completely forgot to merge, sorry. Merging.

**Pull request merged**